api/src/main/java/org/pmiops/workbench/api/CloudTaskEgressController.java

```diff
@@ -3,13 +3,16 @@
 import org.pmiops.workbench.exfiltration.EgressRemediationService;
 import org.pmiops.workbench.model.ProcessEgressEventRequest;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
 import org.springframework.http.ResponseEntity;
 import org.springframework.web.bind.annotation.RestController;
 
 @RestController
 public class CloudTaskEgressController implements CloudTaskEgressApiDelegate {
 
-  @Autowired private EgressRemediationService egressRemediationService;
+  @Autowired
+  @Qualifier("sumologic-egress-service")
```

Comment:
In case of a normal sumologic event, use the sumologic specific implementation of the EgressRemediationService